### ISSUE TITLE
a11y_Pagination> Inverse> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#513]
**Describe the bug**
Pagination>> "1 violations" is appearing for "Inverse". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/pagination--inverse'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for Pagination> Inverse.

**Screenshots**
https://somup.com/c3n2QdZWWk

**Desktop (please complete the following information):**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The issue is fixed.
Storybook Link : https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/pagination--inverse
See Screen-cast : http://somup.com/c3jQfYuJtu
Thanks!

**Comment by Uditsingh67** [comment]
Issue does not exist.
See screen-cast : http://somup.com/c3jvhNUlpu
